File: easy/array_pair_sum.go
package easy

import (
	"slices"
)

func ArrayPairSum(nums []int) int {
	total := 0
	slices.Sort(nums)
	for i := 0; i < len(nums)-1; i += 2 {
		total += nums[i]
	}
	return total
}

File: src/easy/array_pair_sum.rs
pub fn array_pair_sum(nums: Vec<i32>) -> i32 {
    let mut total: i32 = 0;
    let mut arr = nums;
    arr.sort();
    
    for i in (0..arr.len()).step_by(2){
        total += arr[i];
    }
    total
}
